test/spe_test.py

```diff
@@ -78,6 +78,14 @@ def test_send_spanish_score(self):
         response = self.score_handler.send_spanish_score(5.0, 'studenttest')
         self.assertEqual(response.status_code, 200)
 
+    def test_send_spanish_score_without_encoding(self):
```

Comment:
a bit confused by the naming: maybe test_send_spanish_score_with_friend_account()?

Reply:
done